### PR TITLE
Aphrodite JSS deprecation

### PR DESCRIPTION
**Pull request by kebabmaster**
Hi @Rogger794,

I have replaced the **aphrodite-jss* implementation with a custom styling solution that does not require any external dependencies.

Tested locally and with the **gh-pages** branch, works correctly.

The unit tests have been updated accordingly.

No changes in workflows or documentation.

Thank you for your time and have a wonderful day.

### CODE REVIEW

**Comment by aronhelser** [comment]
You're close on the commit message format, but not quite 😄 
`fix(aphrodite_jss_deprecation): aphrodite_jss replaced ...`

**Comment by kebabmaster** [comment]
> You're close on the commit message format, but not quite 
> `fix(aphrodite_jss_deprecation): aphrodite_jss replaced ...`

Argh, supposed to be the easiest part! Fixed.

File: src/utils/uuid.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/**
+* By w3resource, 2020, https://www.w3resource.com/javascript-exercises/javascript-math-exercise-23.php
+*/
+
+export function generateUUID(){
+    var dt = new Date().getTime();
```
Comment (Rogger794) [inline src/utils/uuid.js]: this could be replace by a let variable

File: src/utils/uuid.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    var uuid = "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g, (c) => {
+        var r = (dt + Math.random()*16)%16 | 0;
```
Comment (Rogger794) [inline src/utils/uuid.js]: r should be const

File: src/utils/uuid.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/**
+* By w3resource, 2020, https://www.w3resource.com/javascript-exercises/javascript-math-exercise-23.php
+*/
+
+export function generateUUID(){
+    var dt = new Date().getTime();
+
+    var uuid = "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx".replace(/[xy]/g, (c) => {
```
Comment (Rogger794) [inline src/utils/uuid.js]: should be let variable

File: src/utils/uuid.js (hunk 1 above)
Comment (kebabmaster) [reply src/utils/uuid.js]: Addressed

File: src/utils/uuid.js (hunk 2 above)
Comment (kebabmaster) [reply src/utils/uuid.js]: Addressed

File: src/utils/uuid.js (hunk 3 above)
Comment (kebabmaster) [reply src/utils/uuid.js]: Addressed

**Comment by kebabmaster** [comment]
Additionally, added the **no-var** **eslint** rule that prevents the further contributors from using **var** instead of **const** or **let**.

**Pull request merged**